domain-model.md

```diff
@@ -0,0 +1,22 @@
+# DOMAIN MODEL
+
+## Cohort
+
+Class | Properties | Methods | Notes | Scenario | Output | Example
+----- | ---------- | ------- | ----- | -------- | ------ | -------
+| Cohort | cohortList @Array | createCohort(@String) | new cohort created with a cohort name | valid cohort name input (i.e. string) | return true, cohort added to cohortList | `createCohort("Cohort 1") => true, cohortList = [{name: "Cohort 1", students: []}]` |
+| | | | | invalid cohort name input | throw error | `createCohort(1) => "cohort name must be a valid string"` |
+| | | findCohort(@String) | search for cohort by cohort name | valid & existing cohort name input | return cohort as an object | `findCohort("Cohort 1") => {name: "Cohort 1", students: []}` |
+| | | | | invalid cohort name input | throw error | `findCohort(1) => "cohort does not exist"` |
+| | | addStudentToCohort(@String, @Int) | add student to specific cohort. @String = cohortName, @Int = studentId | valid & existing cohort name and studentId input | return true, cohortList is updated | `addStudentToCohort("Cohort 1", 1) => true, cohortList = [{name: "Cohort 1", students: [{studentId: 1, firstName: "John", lastName: "Doe", githubUsername: "johndoe", email: "john@doe.com"}]}]` |
```

Comment:
have you considered return the changed array instead of true to be able to check its contents?